### DISCUSSION TITLE
카테고리 썸네일 이미지 크기 문제

### DISCUSSION THREAD

**Discussion by junghyeonsu** [pronist/hELLO#79] [Q&A] [answered]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: window 10
- 브라우저: chrome
- 버전: 92.0.4515.107
- 스킨 버전 3.4.3

**설명**
버그는 아니고 블로그 카테고리에서 썸네일의 크기가 너무 커보이는 현상이 있어서
조금 작게 조절이 가능한지에 대해서 피드백을 드립니다.

**스크린샷 및 링크**

### 브라우저 확대 100% 일 때
![image](https://user-images.githubusercontent.com/54893898/127520122-e7554033-7923-4a5d-bc8b-1877ec8781e6.png)

### 브라우저 확대 75% 일 때
![image](https://user-images.githubusercontent.com/54893898/127520209-e8d5360c-8a89-4a71-b576-757ab897ce92.png)

### 브라우저 확대 50% 일 때
![image](https://user-images.githubusercontent.com/54893898/127520278-2b64e57a-b975-4913-a487-2b43b957ebf9.png)

그리고 [제 블로그 링크](https://junghyeonsu.tistory.com/category/%EC%95%8C%EA%B3%A0%EB%A6%AC%EC%A6%98/%ED%94%84%EB%A1%9C%EA%B7%B8%EB%9E%98%EB%A8%B8%EC%8A%A4%202%EB%8B%A8%EA%B3%84)도 걸어드리겠습니다. 스킨 입혀놨어요!

**Comment by pronist** [comment, marked as answer]
스킨 커버, 또는 최신글의 모드가 ```리스트 (오리지널)``` 로 되어있는 경우에 나타나는 기본적인 스킨 커버입니다. ```리스트 (오리지널)``` 의 이미지 사이즈 별도 설정은 직접 수정하지 않는 한 제공하지 않으며 가급적이면 ```리스트 (오리지널)``` 이 아닌 ```리스트``` 와 같은 다른 모드로 스킨을 사용하길 권장하고 있습니다. 이 설정은 **스킨 커버** 또는 **스킨 편집 - 기본 목록 스타일**에서 변경할 수 있습니다. :)

**Comment by junghyeonsu** [comment]
감사합니다! 예쁜 스킨 만들어주셔서 감사합니다 잘 쓸게요!
